src/common/errors.js

```diff
@@ -1,7 +1,18 @@
 'use strict';
 const util = require('util');
 
-class RippleError extends Error {
+// this is needed because extending builtins doesn't work in babel 6.x
+function extendableBuiltin(cls) {
+  function ExtendableBuiltin() {
+    cls.apply(this, arguments);
+  }
+  ExtendableBuiltin.prototype = Object.create(cls.prototype);
+  Object.setPrototypeOf(ExtendableBuiltin, cls);
+
+  return ExtendableBuiltin;
+}
+
+class RippleError extends extendableBuiltin(Error) {
```

Comment:
Ah, I think I know what's happening. Babel 6 doesn't do anything by default. So, the code runs, because I'm running this on node v4. I need to try running this with the transform on